support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep.java

```diff
@@ -85,7 +85,7 @@ public DescriptorImpl() {
         @Override public Step newInstance(StaplerRequest req, JSONObject formData) throws FormException {
             BuildTriggerStep step = (BuildTriggerStep) super.newInstance(req, formData);
             // Cf. ParametersDefinitionProperty._doBuild:
-            JSONArray params = formData.optJSONArray("parameter");
+            JSONArray params = JSONArray.fromObject(formData.get("parameter"));
```

Comment:
Ok. I thought `JSONArray.fromObject` would [throw an exception](https://github.com/jenkinsci/json-lib/blob/master/src/main/jdk15/net/sf/json/JSONArray.java#L189) if the parameter is `null` (java null not JS null).

If the test is actually checking that case, everything should be fine.